plugins/juju/juju.plugin.zsh

```diff
@@ -0,0 +1,134 @@
+# ---------------------------------------------------------- #
+# Aliases and functions for juju (https://juju.is)           #
+# ---------------------------------------------------------- #
+
+# Load TAB completions
+_load_tab_completions() {
+  autoload -U +X compinit && compinit
+  autoload -U +X bashcompinit && bashcompinit
+  local jcomp="$(pkg-config --variable=completionsdir bash-completion)/juju"
+  [ -f "$jcomp" ] && source "$jcomp"
+}
+_load_tab_completions
```

Comment:
You can make a nameless function for it to be automatically called afterwards. This also has the advantage of not polluting the namespace with another function name:
```suggestion
() {
  local completion_file="$(pkg-config --variable=completionsdir bash-completion)/juju"
  [[ -f "$completion_file" ]] && source "$completion_file"
}
```